### PR TITLE
Create autonomyType.md

### PR DESCRIPTION
**Pull request by rgopikrishnan91**
create a properties file for autonomyType @zvr @kestewart @lumjjb

### CODE REVIEW

File: model/AI/Properties/autonomyType.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# autonomy
```
Comment (maxhbr) [inline model/AI/Properties/autonomyType.md:3]: the title needs to be consistent:

```suggestion
# autonomyType
```

**Review by maxhbr — CHANGES_REQUESTED** [review]

**Review by kestewart — CHANGES_REQUESTED** [review]
Fix capitalization of  autonomyType to AutonomyType, to be consistent.

**Comment by rgopikrishnan91** [comment]
It should still be autonomyType --since all fields in properties are named with camel case (where first letter is lower case), for the data types defined in Vocabulary are named with camel case (but the first letter also being in upper case). autonomyType is a property of the data type (PresenceType).

@zvr could you please confirm this rationale?

**Review by bennetkl — APPROVED** [review]
Looks good.

**Pull request merged**